### PR TITLE
Add link to download html zip if available

### PR DESCRIPTION
**Pull request by nrdg42**
*Issue #, if available:*

*Description of changes:*
* Add link to download html zip if available
* Move run.json link from footer to new downloads area

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: templates/dashboard.jinja.html [hunk 1]
```diff
@@ -361,6 +390,25 @@ <h1>
...
+          href="run.json">run.json</a></code>
+        </p>
+      </li>
+      {% if litani_zip_path is not none %}
+      <li>
+        <p>HTML report: <code><a
+          href={{litani_zip_path}} download>{{litani_zip_path}}</a></code>
+        </p>
+      </li>
+      {% endif %}
```
Comment (karkhaz) [inline templates/dashboard.jinja.html]: Could we follow the
```
{% endif %}{# litani_zip_path is not none #}
```
here, these blocks eventually get quite hard to follow

File: templates/dashboard.jinja.html [hunk 2]
```diff
@@ -361,6 +390,25 @@ <h1>
...
+      <li>
+        <p>
+          Raw data for this run: <code><a
+          href="run.json">run.json</a></code>
+        </p>
+      </li>
+      {% if litani_zip_path is not none %}
+      <li>
+        <p>HTML report: <code><a
+          href={{litani_zip_path}} download>{{litani_zip_path}}</a></code>
```
Comment (karkhaz) [inline templates/dashboard.jinja.html]: Jinja style is to have whitespace within expression and statements: `{{ litani_zip_path }}` rather than `{{litani_zip_path}}`. I prefer this, it makes it standout from the text

Comment (karkhaz) [reply templates/dashboard.jinja.html]: `href={{litani_zip_path}} download>` looks incorrect, do you mean `href={{litani_zip_path}}>download`?

File: lib/litani_report.py [hunk 3]
```diff
@@ -310,10 +310,12 @@ def render(run, report_dir):
 
     svgs = render_runtimes(run, env, report_dir)
 
+    litani_zip_path = os.environ.get("LITANI_ZIP_PATH")
```
Comment (karkhaz) [inline lib/litani_report.py]: This is a bit generic, it'll get confusing if we have other zip files in the future (which I expect we will). Can I suggest `LITANI_REPORT_ARCHIVE_PATH`?

**Review by karkhaz — CHANGES_REQUESTED** [review]

File: lib/litani_report.py (hunk 3 above)
Comment (karkhaz) [reply lib/litani_report.py]: Also `os.getenv` is shorter than `os.environ.get`

File: templates/dashboard.jinja.html (hunk 2 above)
Comment (karkhaz) [reply templates/dashboard.jinja.html]: And actually to be good HTML it should really be `href="{{ litani_zip_path }}">download`

Comment (nrdg42) [reply templates/dashboard.jinja.html]: Download is actually an [anchor tag attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a). Let me know if that is not what we want here.

Comment (karkhaz) [reply templates/dashboard.jinja.html]: Oh, very cool! Thanks! But I still think there should be quotes around the href `href="{{ litani_zip_path }}"`

**Review by karkhaz — APPROVED** [review]
